src/Utilities/src/MoistThermodynamics.jl

```diff
@@ -400,20 +400,11 @@ The optional input value of the temperature `T_init` is taken as the initial
 value of the saturation adjustment iterations.
 """
 function saturation_adjustment(e_int, ρ, q_t, T_init = T_triple)
-    tol_abs = 1e-3*cv_d
-    iter_max = 10
-    args = (ρ, q_t, e_int)
-    T0 = max(T_min, air_temperature(e_int, q_t, 0.0, 0.0))
-    T1 = air_temperature(e_int, q_t, 0.0, q_t)
-    roots_equation(x, ρ, q_t, e_int) = internal_energy_sat(x, ρ, q_t) - e_int
-    T, converged = find_zero(roots_equation,
-                             T0, T1,
-                             args,
-                             IterParams(tol_abs, iter_max),
-                             SecantMethod()
-                             )
-    return T
-
+  T0 = max(T_min, air_temperature(e_int, q_t, 0.0, 0.0))
+  T1 = air_temperature(e_int, q_t, 0.0, q_t)
+  T, converged = find_zero(x -> internal_energy_sat(x, ρ, q_t) - e_int,
```

Comment:
We should set a lower `maxiter` so that this function call does not hang. I think that Tapio suggested `maxiter = 10`. I almost wonder if this parameter should be passed in as an argument so that we can worry about it upstream.

Reply:
Probably, I was just trying to keep this consistent.

Do you know if we could use Newton iterations for this?

Reply:
I think that we _could_, but I think that one advantage of the Secant method is that we've got a good idea of realistic physical bounds. That said, we can always implement both and see which one works better. I've typically used Newton iteration in the past and found it effective.

Reply:
Secant has been more reliable for us, at essentially the same computational cost as Newton.

Reply:
We don't want more than ~10 iterations (and can have larger relative tolerances, see below).